lms/models/organization.py

```diff
@@ -0,0 +1,80 @@
+from base64 import urlsafe_b64encode
+from uuid import uuid4
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+
+from lms.db import BASE
+from lms.models._mixins import CreatedUpdatedMixin
+from lms.models.application_settings import ApplicationSettings
+from lms.models.region import Region
+
+
+class Organization(CreatedUpdatedMixin, BASE):
+    """Model for Organizations comprised of application instances."""
+
+    __tablename__ = "organization"
+
+    id = sa.Column(sa.Integer(), autoincrement=True, primary_key=True)
+
+    _vanity_id = sa.Column(
+        sa.UnicodeText(),
+        nullable=False,
+        unique=True,
+        # We don't use a standard UUID-4 format here as they are common in Tool
+        # Consumer Instance GUIDs, and might be confused for them. These also
+        # happen to be shorter and guaranteed URL safe.
+        default=lambda: urlsafe_b64encode(uuid4().bytes).decode("ascii").rstrip("="),
+    )
+    """
+    A human readable URL safe vanity id.
+
+    The vanity id can be customised by the user and therefore can end-up not
```

Comment:
> The vanity id can be customized by the user

Is this true? I thought of this ID as set in stone as the primary key.

Would you then expect the value to be for example editable on the admin pages?